src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/PossibleOperationValidator.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.sandl.snlevents.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.util.function.BiFunction;
+import java.util.function.Function;
+
+@Data
+@AllArgsConstructor
+public class PossibleOperationValidator {
+    private Function dbConfigVerifier;
```

Comment:
why not Function<StatusConfig, Boolean>?
plus Class names for generics in BiFunction?